_posts/2015-07-06-code-reviews-for-learning.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Why Code Reviews Are Great Learning Tools"
+description: "Code reviews are a good way to make sure (or at least, somewhat more sure) that crapp or buggy code does not make its way to production. But, code reviews are also a terrific way to consolidate your learning. Here's why!"
+tags: ["learning", "resources", "hackership"]
+authors:
+ - anouk
+image: /content/2015-07-06-code-review-whiteboard.jpg
+---
+
+###What is a code review?
+
+A code review is the examination of your code and feedback on your code by a peer, or coach. Code reviews can take place online (for instance by using Github), or in person – offline.
+
+![code reviews are awesome](/content/2015-07-06-code-review-whiteboard.jpg)
+
+###How does a code review work?
+
+Finished a piece of code (or a feature)? Create a patch or pull request and send or assign this to a peer or coach. They will look at your code and make comments, add questions, or give suggestions. 
+
+Alternatively, you could walk someone through your code and explain them what is going on line by line. This is aptly called a code walkthrough.
```

Comment:
a walkthrough can also be on an architectural level – it doesn't have to be line by line (which is often to exhaustive).